VogtPayroll2/Employee1.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    abstract class Employee
+    {
+        //For simplicity, use a constant tax rate of 30% to compute the tax amount
+        protected string name;
+        protected int hoursWorked;
+        protected decimal payRate;
+        protected const decimal taxAmount = .30m;
+
+        public Employee(string aName, int aHoursWorked, decimal aPayRate)
```

Comment:
Don't collect the `payRate` as a parameter here. You don't need it in the base class. Instead, you'll call `GetPayRate()` when you need it.